micmac_li3ds/import_autocal.py

```diff
@@ -88,19 +88,27 @@ def take_action(self, parsed_args):
 
         root = xmlutil.root(self.filename, 'ExportAPERO')
         node = xmlutil.child(root, 'CalibrationInternConique')
+        assert(xmlutil.child(node, 'KnownConv').text.strip()
+               == 'eConvApero_DistM2C')
```

Comment:
Same comment on using `assert` versus raising an exception with a specific message. No strong opinion about this, I'd rather provide a clear message to the user when the XML document is not as expected.